docs/mpc/mpc.rst

```diff
@@ -0,0 +1,103 @@
+.. doctest-skip-all
+
+.. _astroquery.mpc:
+
+**********************************************
+Minor Planet Center Queries (`astroquery.mpc`)
+**********************************************
+
+Getting started
+===============
+
+This is an Astroquery wrapper for querying the
+`Minor Planet Center <https://minorplanetcenter.net/web_service//>`__
+web service, which returns the pure JSON response from the service.
+
+The MPC web service allows for querying by any of the parameters listed in the function headers in
+mpc/core.py, and also allows for querying for a range of values.
+
+Available methods
```

Comment:
Drop this section, you already have an autogenerated (and thus always up-to-date) list of the methods at the bottom of the document.